paasta_tools/mesos_tools.py

```diff
@@ -655,21 +655,10 @@ def terminate_framework(framework_id):
     resp.raise_for_status()
 
 
-def get_container_name(app_id, slave_hostname=None, task_id=None):
+def get_tasks_from_app_id(app_id, slave_hostname=None, task_id=None):
```

Comment:
I don't really understand why you've moved ``get_container_name`` to be in ``cli/utils.py``? Its usage is wider reaching that just the cli tools.